Instagram/IGListKit
# Add tvOS support. Close #118

## Changes in this pull request

Closes #118. 

- Add tvOS framework + test targets
- Add shared scheme `IGListKit-tvOS`
- Update `Podfile` and `.podspec`
- Update `.travis.yml`
- Update `CHANGELOG`
- Update `README`
- Run `pod install` and check-in updates

## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/CONTRIBUTING.md)
